packages/accounts-controller/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@metamask/accounts-controller",
+  "version": "1.0.0",
+  "description": "Manages internal accounts",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/accounts-controller#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build:docs": "typedoc",
+    "changelog:validate": "../../scripts/validate-changelog.sh @metamask/accounts-controller",
+    "publish:preview": "yarn npm publish --tag preview",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@metamask/base-controller": "^3.2.1",
+    "@metamask/eth-snap-keyring": "^0.2.2",
+    "@metamask/keyring-api": "^0.2.5",
+    "@metamask/keyring-controller": "^7.4.0",
+    "@metamask/snaps-controllers": "^1.0.1",
+    "@metamask/snaps-utils": "^1.0.1",
+    "@metamask/utils": "^6.2.0",
+    "eth-rpc-errors": "^4.0.2",
+    "ethereumjs-util": "^7.0.10",
+    "immer": "^9.0.6",
+    "nanoid": "^3.1.31",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "@metamask/auto-changelog": "^3.1.0",
+    "@types/jest": "^27.4.1",
+    "@types/readable-stream": "^2.3.0",
+    "deepmerge": "^4.2.2",
+    "jest": "^27.5.1",
+    "sinon": "^9.2.4",
+    "ts-jest": "^27.1.4",
+    "typedoc": "^0.22.15",
+    "typedoc-plugin-missing-exports": "^0.22.6",
+    "typescript": "~4.6.3"
+  },
+  "peerDependencies": {
+    "@metamask/keyring-controller": "^7.4.0"
```

Comment:
We can probably remove this and leave it in `devDependencies`

Reply:
I might have given a wrong suggestionhere: 
Seeing how we use messenger events/actions types in other packages, we usually include the package they come from in `dependencies` and `peerDependencies`, probably to ensure that we use the same versions of that package across the dep tree

@Gudahtt might have more insights on this, but I think we should rollback these changes - sorry about that.

Reply:
no problem, the change has been d5e3b8e195f043553b171dcdf8b712d993608a45